recipes/cmaes/all/patches/CompilerOptionsEigen.patch

```diff
@@ -0,0 +1,142 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 4a38212..dccfea4 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
```

Comment:
I see this comes from https://github.com/CMA-ES/libcmaes/pull/234, do you have any insight into wether it will be merged eventually?

Reply:
I hope so. I will trigger @beniz again. If there is a new version with the PR, then I can adjust the conanfile here to not include this patch for the new version